packs/magento.json

```diff
@@ -1,23 +1,23 @@
 {
   "unused_css_rules": "Magento does not have built-in support for deferring CSS, but it can be accomplished using third-party tools in CI or in local development.",
   "uses_webp_images": "Consider searching the [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=webp) for a variety of third-party extensions to leverage newer image formats.",
-  "offscreen_images": "Consider searching the [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=lazy%20load) for a third-party extension to offscreen lazy-load images and products.",
-  "total_byte_weight": "Disable the built-in bundling feature in Magento, and instead utilize [r.js bundling](https://github.com/magento/m2-devtools/blob/master/docs/panels/RequireJS.md).",
-  "render_blocking_resources": "",
-  "unminified_css": "",
-  "unminified_javascript": "",
+  "offscreen_images": "Consider modifying your product and catalog templates to make use of the web platform's [lazy loading](https://web.dev/native-lazy-loading) feature",
+  "total_byte_weight": "Disable Magento's built-in JavaScript bundling and minification, and consider using [baler](https://github.com/magento/baler/)",
+  "render_blocking_resources": "Disable Magento's built-in JavaScript bundling, and consider using [baler](https://github.com/magento/baler/) instead",
+  "unminified_css": "Enable the \"Minify CSS Files\" option in the store's Developer settings",
+  "unminified_javascript": "Use [Terser](https://www.npmjs.com/package/terser) to minify all JavaScript assets outfrom from static content deployment, and disable the built-in minification feature",
   "efficient_animated_content": "",
-  "unused_javascript": "",
+  "unused_javascript": "Disable Magento's built-in JavaScript bundling",
```

Comment:
Any chance we have a link to a doc page/instructions for this?

Reply:
If not no worries, I'll link to [this](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/js-bundling.html) if there's no specific page that describes disabling

Reply:
Nothing at the moment, but the plan is to get these features ripped out in a major release (they're old and counter productive)